src/utils/helpers.ts

```diff
@@ -259,3 +292,22 @@ export function getImgProxyUrl(originalUrl?: string) {
 
   return `${proxyUrl}/plain/${originalUrl}`;
 }
+
+export function sortSolanaTokens(solanaTokens: SolanaToken[], sortType: SORT_SPL_TOKEN, currency = "usd") {
```

Comment:
why need for different currency ?
usd should be enough to determine the price.

why this function is needed?

Reply:
does the currency parameter needed as any currency would not affect the sorting